### PR TITLE
User can create profile and user can add friends without duplicate friends.

### PR DESCRIPTION
**Pull request by aidanbdh**
I had to fix the bug (#30) in order to properly test creating a profile

### CODE REVIEW

File: createProfile.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+var $work = document.getElementById('new-work');
+var $school = document.getElementById('new-school');
+var $home = document.getElementById('new-home');
+var $interests = document.getElementById('new-interests');
+var $bio = document.getElementById('new-bio');
+var $quotes = document.getElementById('new-quotes');
+var $submit = document.getElementById('profile-button');
+
+var createProfile = function() {
+<<<<<<< HEAD
```
Comment (thebearingedge) [inline createProfile.js]: Is this still in your source code?

File: default.css [hunk 2]
```diff
@@ -102,12 +102,55 @@ body {
...
+  border-width: 8px;
+  border-color: rgba(255,255,255,0);
+  border-bottom-color: rgb(255,255,255);
+  z-index: 3;
+}
+
+#login,
+#signup {
   float: right;
+>>>>>>> 62e91d12320a01a40ced9395cd6794108a287bbd
```
Comment (thebearingedge) [inline default.css]: Looks like you still have some Merge Conflict artifacts in here.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

File: default.css (hunk 2 above)
Comment (aidanbdh) [reply default.css]: Fixed

File: createProfile.js (hunk 1 above)
Comment (aidanbdh) [reply createProfile.js]: It was but not anymore I fixed the conflict

**Pull request merged**